frame/nfts/src/features/create_delete_item.rs

```diff
@@ -123,27 +120,28 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 			item_config,
 			|_, _| Ok(()),
 		)?;
-		let origin = Self::find_account_by_role(&collection, CollectionRole::Admin)
-			.unwrap_or(collection_details.owner.clone());
-		for (key, value) in attributes {
-			Self::do_set_attribute(
-				origin.clone(),
-				collection,
-				Some(item),
-				AttributeNamespace::CollectionOwner,
-				Self::construct_attribute_key(key)?,
-				Self::construct_attribute_value(value)?,
-				mint_to.clone(),
-			)?;
-		}
-		if !metadata.len().is_zero() {
-			Self::do_set_item_metadata(
-				Some(origin.clone()),
-				collection,
-				item,
-				metadata,
-				Some(mint_to.clone()),
-			)?;
+		let admin_account = Self::find_account_by_role(&collection, CollectionRole::Admin);
+		if let Some(admin_account) = admin_account {
```

Comment:
just wondering if admin_account is_none && we are root that ought to be allowed but I'm not sure that root would ever be interested in doing so?

Reply:
root won't be able to pre-sign anything, and there are other methods for the root account like force_mint()